include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -54,6 +54,17 @@ namespace aspect
         plastic_weakening_with_plastic_strain_and_viscous_weakening_with_viscous_strain,
         viscous_weakening_with_viscous_strain_only
       };
+      /**
+       * Enumeration for selecting which type of healing mechanism to use.
+       * For none, no strain healing occurs.
+       * Otherwise, the material is healed based on a temperature dependent
+       * strain healing calculation
```

Comment:
```suggestion
       * Future models could consider strain healing formulations that are a function
       * of time, deformation rate, or other parameters.
```

Reply:
Given that the temperature dependent model effectively follows the Frank Kamenetskii viscosity formulation, I'm wondering if we should rename `temperature_dependent` to something like `frank_kamenetskii_temperature_dependent`?